### ISSUE TITLE
Add const file for string

### ISSUE THREAD

**Issue by codeithuman** [infinitered/firebird#11]
How should we manage strings within our phoenix applications? Flash messages, error messages, validation messages, etc.

**Comment by ryanlntn** [comment]
If these are user facing strings then I think we should go with some type of internationalization. https://hexdocs.pm/gettext/Gettext.html Seems to be what Phoenix includes by default.

More info: https://medium.com/@erichkist/using-gettext-to-internationalize-a-phoenix-application-ceb410407ea5

**Comment by silasjmatson** [comment]
@codeithuman Jumping on to what @ryanlntn said, I think I18n is the route we should go.

I think we should consider using [linguist](https://github.com/chrismccord/linguist) instead of Gettext though.

**Comment by ryanlntn** [comment]
Totally down with linguist too. I'd prefer to work in plain elixir code. There are some advantages to Gettext though like the `mix gettext.extract --merge` task to automatically sync all existing entries to .pot template files and merge them into locale-specific .po files. As far as I know linguist doesn't support this.

**Comment by silasjmatson** [comment]
Yeah, linguist is more of a key-value store than a full-on I18n solution.
